en/advanced_features/precland.md

```diff
@@ -70,29 +74,28 @@ If the beacon is still not visible at the search altitude and after a search tim
 In *Opportunistic Mode* the vehicle will use precision landing *if* (and only if) the beacon is visible when landing is initiated.
 If it is not visible the vehicle immediately performs a *normal* landing at the current position.
 
-## Performing a Precision Landing
+## Initiating a Precision Landing
 
 :::note
-Due to a limitation in the current implementation of the position controller, precision landing is only possible with a valid global position.
+Precision landing is only possible with a valid global position (due to a limitation in the current implementation of the position controller).
 :::
 
-### Via Command
-
-Precision landing can be initiated through the command line interface with
-```
-commander mode auto:precland
-```
-In this case, the precision landing is always considered "required".
-
 <span id="mission"></span>
 ### In a Mission
 
 Precision landing can be initiated as part of a [mission](../flying/missions.md) using [MAV_CMD_NAV_LAND](https://mavlink.io/en/messages/common.html#MAV_CMD_NAV_LAND) with `param2` set appropriately:
 
-- `param2` = 0: Normal landing without using the beacon.
-- `param2` = 1: *Opportunistic* precision landing.
-- `param2` = 2: *Required* precision landing.
+- `0`: Normal landing without using the beacon.
+- `1`: [Opportunistic](#opportunistic-mode) precision landing.
+- `2`: [Required](#required-mode) precision landing.
 
+### Command Line
```

Comment:
On further thought, this should probably be NuttX or SITL console. 

This is a bit poor usability. Yes I can go in through the mavlink console. Presumably I could also use MAVLink Set_Mode message (but what would I specify for the custom and main mode?). But much better would be to enable the MAV_CMD_NAV_LAND mission item as a command rather than just mission item. Any chance of this?

Reply:
Man this is confusing. I know for a fact that if the precision landing mode paramter `RTL_PLD_MD` is set to 1 or 2, it will perform the precision landing also during a mission in place of a standard RTL.

I'm actually not sure about the existing text here with mavlink and param2...

Reply:
The simplest way of using precision landing is really to configure parameter `RTL_PLD_MD` and then just execute "RTL", which will internally do the precision landing. That's why the above mention of a "Precision Landing" flight mode also confused me.